modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/comments/DevPortalCommentTest.java

```diff
@@ -169,134 +172,143 @@ In some databases(mysql, oracle), the timestamp format do not store milliseconds
         for (CommentDTO replyDTO : replyListDTO.getList()) {
             assertEquals(replyDTO.getCategory(), "general", "Comments do not match");
             assertEquals(replyDTO.getEntryPoint(), CommentDTO.EntryPointEnum.DEVPORTAL, "Comments do not match");
             assertEquals(replyDTO.getParentCommentId(), rootCommentIdToAddReplies, "Comments do not match");
             assertEquals(replies.contains(replyDTO.getId()), true, "Comments do not match");
         }
+    }
 
+    @Test(dependsOnMethods = {"testDevPortalGetRepliesOfCommentTest"}, groups = { "wso2.am" }, description = "Verify Pagination of Replies List of Comment Test Case")
+    public void testDevPortalPaginationOfRepliesOfCommentTest() throws Exception {
         // Verify Pagination of replies list of a comment
-        HttpResponse getRepliesToVerifyPagination = restAPIStore.getReplies(rootCommentIdToAddReplies, apiId, gatewayContextWrk
-                .getContextTenant().getDomain(), false, 3, 1);
+        HttpResponse getRepliesToVerifyPagination = restAPIStore
+                .getReplies(rootCommentIdToAddReplies, apiId, gatewayContextWrk.getContextTenant().getDomain(), false, 3, 1);
         assertEquals(getRepliesToVerifyPagination.getResponseCode(), Response.Status.OK.getStatusCode(),
                 "Error retrieving comment");
         Gson getRepliesToVerifyPaginationGson = new Gson();
-        CommentListDTO getRepliesToVerifyPaginationCommenListDTO = getRepliesToVerifyPaginationGson.fromJson(getRepliesToVerifyPagination
-                .getData().replace("devportal", "DEVPORTAL"), CommentListDTO.class);
+        CommentListDTO getRepliesToVerifyPaginationCommenListDTO = getRepliesToVerifyPaginationGson
+                .fromJson(getRepliesToVerifyPagination.getData().replace("devportal", "DEVPORTAL"), CommentListDTO.class);
         assertEquals(getRepliesToVerifyPaginationCommenListDTO.getList().size(), 2, "Comments limit does not match");
-        assertEquals(getRepliesToVerifyPaginationCommenListDTO.getList().get(0).getContent(), "This is a reply 2", "Offset value does not captured");
-        assertEquals(getRepliesToVerifyPaginationCommenListDTO.getList().get(1).getContent(), "This is a reply 3", "Offset value does not captured");
+        assertEquals(getRepliesToVerifyPaginationCommenListDTO.getList().get(0).getContent(), "This is a reply 2",
+                "Offset value does not captured");
+        assertEquals(getRepliesToVerifyPaginationCommenListDTO.getList().get(1).getContent(), "This is a reply 3",
+                "Offset value does not captured");
+    }
 
-        //Edit a comment
-        //Edit the content only
-        HttpResponse editCommentResponse = restAPIStore.editComment(rootCommentIdToAddReplies, apiId,
-                "Edited root comment", "general");
+    @DataProvider (name = "input-data-provider")
+    public Object[][] inputDataProviderMethod(){
```

Comment:
Shall we fix this formatting error?

Reply:
I will send this change with https://github.com/wso2/product-apim/pull/11163 PR

Reply:
Fixed with https://github.com/wso2/product-apim/pull/11163/commits/909e61ea2edc8a1e3f88aead2a5213324d94bd84